contracts/contracts/token/OUSD.sol

```diff
@@ -48,6 +56,7 @@ contract OUSD is Initializable, InitializableToken, Governable {
 
         _totalSupply = 0;
         totalCredits = 0;
+        nonRebasingSupply = 0;
```

Comment:
Note: initialize is not going to get called when we upgrade the OUSD contract on mainnet. But that's probably ok since by default in solidity variables are set to 0 I believe?

For consistency, should we set nonRebasingSupply to zero as well here?

Reply:
nonRebasingCredits you mean right? Will do 👍